etl/storage/storage.go

```diff
@@ -23,5 +24,35 @@ func NewStorage(db *gorm.DB) interfaces.Storage {
 }
 
 func (s *storage) SaveJetDropData(jetDrop models.JetDrop, records []models.Record) error {
-	panic("not implemented")
+	return s.db.Transaction(func(tx *gorm.DB) error {
+		// create zero pulse and zero jetDrop for FK at Record table
+		// TODO: save pulse correctly at PENV-266
+		pulse := models.Pulse{PulseNumber:1}
+		if err := tx.Save(&pulse).Error; err != nil {
+			return errors.Wrap(err, "error while saving pulse")
+		}
+		// TODO: save jetDrop correctly at PENV-267
+		jetDrop.PulseNumber = 1
+		jetDrop.JetID = []byte{1}
+		if err := tx.Save(&jetDrop).Error; err != nil {
+			return errors.Wrap(err, "error while saving jetDrop")
+		}
+
+		for _, record := range records {
+			// TODO: dont rewrite data at record, fix it at PENV-266 and PENV-267
+			record.PulseNumber = 1
+			record.JetID = []byte{1}
+			if err := tx.Save(&record).Error; err != nil {
+				return errors.Wrap(err, "error while saving record")
+			}
+		}
```

Comment:
does tx.Save commit in the DB immediately?  if yes, I think it will be better to save records in some transaction and do commit after the for function, or after  N records.   WDUT?

Reply:
We do every `Save` inside `db.Transaction` func (line 27), so we do commit only after all records and jetDrop will be correctly saved at db.